src/plays/states/States.jsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from "react";
+import { useLocation } from "react-router-dom";
+import { getPlayById } from "meta/play-meta-util";
+import PlayHeader from "common/playlists/PlayHeader";
+import "./States.css";
+
+function States() {
+  // Do not remove the below lines.
+  // The following code is to fetch the current play from the URL
+  const location = useLocation();
+  const { id } = location.state;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+  const [name, setName] = useState("");
+  const [display, setDisplay] = useState(false);
+  const [duration, setDuration] = useState(0);
+
+  const handleSubmit = () => {
```

Comment:
As it is a submit type button, please do a e.preventDefault

Reply:
Done!